WordPress/src/main/java/org/wordpress/android/ui/media/MediaAddFragment.java

```diff
@@ -198,7 +198,10 @@ private String getRealPathFromContentURI(Uri contentUri) {
             return null;
         }
 
-        cursor.moveToFirst();
+        if (!cursor.moveToFirst()) {
+            return null;
```

Comment:
This fixes the problem, but it also leaves the cursor open (there's no call to `cursor.close()`).